packages/gatsby-cli/src/create-cli.js

```diff
@@ -166,8 +166,10 @@ function buildLocalCommands(cli, isLocalSite) {
     builder: _ =>
       _.option(`prefix-paths`, {
         type: `boolean`,
-        default: false,
-        describe: `Build site with link paths prefixed (set pathPrefix in your gatsby-config.js).`,
+        default: process.env.PREFIX_PATHS || false,
```

Comment:
Should we do the env dance?
```suggestion
        default: process.env.PREFIX_PATHS === 'true' || process.env.PREFIX_PATHS === '1',
```